apps/hash-graph/lib/graph/src/store/postgres/query/table.rs

```diff
@@ -886,11 +975,11 @@ pub enum Column<'p> {
     EntityEditions(EntityEditions<'p>),
     PropertyTypeConstrainsValuesOn(PropertyTypeConstrainsValuesOn),
     PropertyTypeConstrainsPropertiesOn(PropertyTypeConstrainsPropertiesOn),
-    EntityTypeConstrainsPropertiesOn(EntityTypeConstrainsPropertiesOn),
-    EntityTypeInheritsFrom(EntityTypeInheritsFrom),
-    EntityTypeConstrainsLinksOn(EntityTypeConstrainsLinksOn),
-    EntityTypeConstrainsLinkDestinationsOn(EntityTypeConstrainsLinkDestinationsOn),
-    EntityIsOfType(EntityIsOfType),
+    EntityTypeConstrainsPropertiesOn(EntityTypeConstrainsPropertiesOn, u32),
```

Comment:
I know this is the inheritance depth, but this might not be so clear to a person new to the codebase. Maybe switch to a struct variant or employ the newtype pattern?

Reply:
Sounds good to me